Navigation tests

Mingling94 opened the pull request.

DavidVorick commented on site/markup/pages/files.haml:
```diff
@@ -27,6 +27,7 @@
                 .no
                     %i.fa.fa-refresh.fa-spin
             .name
+            .size
```
this doesn't cause overflow issues?

DavidVorick replied:
just checked, it's fine for me, but I'm a bit surprised.

lukechampine replied:
there was already some markup for `.size` for some reason, `width: 10%` IIRC

DavidVorick replied:
heh

Mingling94 committed:
Test for navigation

Mingling94 committed:
Shortened navigation test code

Added custom command showPlugin

Also removed js lines that edited webview css when navigating to that
plugin, instead they add a class
